checker/tests/index/Loops.java

```diff
@@ -49,4 +49,21 @@ public void test4(int[] src) {
         int[] optoSft = new int[patternLength];
         for (int i = patternLength; i > 0; i--) {}
     }
+
+    public void test5(
+            boolean flag,
+            @LTLengthOf(value = "#1", offset = "-1000") int offset,
```

Comment:
Shouldn't it be illegal to use the boolean first parameter as value of `LTLengthOf`?